docs/webmin.md

```diff
@@ -0,0 +1,18 @@
+## Description
+This script installs the Webmin web-based administration interface.
+
+## Installation
+```
+$ sudo hassbian-config install webmin
+```
+
+## Upgrade
+```
+$ sudo hassbian-config upgrade webmin
+```
+
+## Additional info
+Default port: 10000
+
```

Comment:
Need more info.
- Where is logging done?
- Are there an config file/dir?
- Are there any service commands that could be useful?

Reply:
Will add them when I get home.

Reply:
Will add this soon